Overview.src.html

```diff
@@ -3029,9 +3036,18 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  <span title=concept-header>headers</span>.
  <!-- XXX xref partial, modify, resume headers -->
 
- <li><p>If <var>response</var> is null, set <var>response</var> to the result of making an
- <span title=concept-http-network-fetch>HTTP-network fetch</span> using <var>httpRequest</var> with
- the <i title>credentials flag</i> set if set.
+ <li>
+  <p>If <var>response</var> is null, run these substeps:
+
+  <ol>
+   <li><p>If <var>httpRequest</var>'s <span title=concept-request-cache-mode>cache mode</span> is
+   "<code title>only-if-cached</code>", then return a
+   <span title=concept-network-error>network error</span>.
```

Comment:
The difference would be that it would not surface as a rejection in the API meaning you'd have to inspect the response status code to know what's going on (which could theoretically be the same).